### ISSUE TITLE
Experimental setup issues for mask pre-training

### ISSUE THREAD

**Issue by ychuest** [qiaoqiaoLF/MxDNA#10]
In the paper, 15% of the tokens are randomly masked. So is there a 10% probability that within that 15% of masked tokens are replaced with randomized tokens?

**Comment by qiaoqiaoLF** [comment]
You are correct. Based on your statement and the reference to Appendix A.4.1, the answer to your question is yes.
